FirestoreWrite.ts

```diff
@@ -48,8 +48,15 @@ class FirestoreWrite {
       if (!maskData.length) {
         throw new Error('Missing fields in Mask!');
       }
-      for (const field of maskData) {
-        request.addParam('updateMask.fieldPaths', `\`${field.replace(/`/g, '\\`')}\``);
+
+      if (typeof mask === 'boolean') {
+        for (const field of maskData) {
+          request.addParam('updateMask.fieldPaths', `\`${field.replace(/`/g, '\\`')}\``);
```

Comment:
Why not set ``\`${field.replace(/`/g, '\\`')}\``` or `field` to a variable based on the if condition?

Reply:
AFAIK, hard to create a replacing pattern outside the 'for loop' because it needs to refer to the 'field'.
Also don't want to do if...else... in each iteration.
I appreciate it if you can share some thoughts.

Reply:
Oh right! I totally was complacent on that one. I'm with you there.

Reply:
This breaks the backward compatibility, since it offsets the escape process for the developer. It also doesn't add support for using ```update(docPath, {'nested.path': updatedValue}, true)``` which could be a valid syntax, I would recommend adding a new parameter to the function that would be called `raw` or something and if this parameter is set, no escaping would be applied.